src/Microsoft.DotNet.Build.Tasks/PackageFiles/msbuild.cmd

```diff
@@ -1,2 +1,2 @@
-@call msbuild.exe %*
+@call %~dp0dotnetcli\dotnet.exe msbuild %*
```

Comment:
Yup that is the plan, this is just temporary. I also plan to do msbuild specific parameters in here( like nodeReuse) cc @maririos